Library/OcAppleKernelLib/CachelessContext.c

```diff
@@ -945,6 +953,19 @@ CachelessContextAddKext (
         if (!Context->Is32Bit) {
           InfoPlistLibraries = InfoPlistLibraries64;
         }
+      } else if ((BundleVersion != NULL) && (AsciiStrCmp (TmpKeyValue, INFO_BUNDLE_VERSION_KEY) == 0)) {
```

Comment:
You do comparison as part of non-debug code, this is undesired.

Reply:
https://github.com/acidanthera/OpenCorePkg/commit/d107500d4566e6ac328b94e12ce843074c3a5a22